src/Http/Exception/HttpException.php

```diff
@@ -30,4 +30,44 @@ class HttpException extends Exception
      * @inheritDoc
      */
     protected $_defaultCode = 500;
+
+    /**
+     * @var array
+     */
+    protected $headers = [];
+
+    /**
+     * Sets http response header.
+     *
+     * @param string $header Header name
+     * @param string|null $value Header value
```

Comment:
Header values can be an array.

Reply:
I forgot about that this time. Updated type hints and changed a test to pass an array of values.